Jenkinsfile

```diff
@@ -13,16 +13,13 @@ properties([
 ])
 
 def buildTypes = ['Linux', 'Windows']
-def jdks = [8, 11, 17]
+def jdks = [11, 17]
```

Comment:
Note: https://github.com/jenkinsci/jenkins/blob/e9d36a0f2eeb9826841a59e34ce5d7a84a022a38/Jenkinsfile#L28 _could_ also be simplified at this time, as in https://github.com/jenkinsci/bom/pull/1118. (Presumably copied from https://github.com/jenkins-infra/pipeline-library/blob/f9a4eec84ee28c7305f2ab629fbb0dd950bca3e7/vars/buildPlugin.groovy#L43 or vice-versa.)